src/style.css

```diff
@@ -0,0 +1,165 @@
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  position: relative;
+  left: 5px;
+  margin-top: 15px;
+  width: 100%;
+  top: 3px;
+}
+
+.search-form__input {
+  height: 42px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+  padding-left: 15px;
+  padding-right: 43px;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.search-form__label {
+  position: relative;
+}
+
+.search-form__label:after {
+  content: "";
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 16px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+}
+
+.search-form__label_big:before {
+  content: "";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 1px;
+  height: 19px;
+}
+
+.search-form__label_big:after,
+.search-form__label_big:before {
+  content: "";
+  display: block;
+}
+
+.search-form__label_big:after {
+  content: "";
+  position: absolute;
+  top: 10px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+  border-radius: 0;
+  box-sizing: border-box;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: top left top;
+}
+
+.search-form__input::placeholder {
+  color: inherit;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  font-style: heavy;
+}
+
+.search-form__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  font-weight: 600;
+}
+
+.search-form_small {
+  margin-left: -2px;
+}
+
+.search-form__input_big {
+  height: 75px;
+  padding-left: 60px;
+}
+
+.search-form_big {
+  margin-left: -5px;
+  margin-top: 15px;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  background: linear-gradient(#e1e7ed, #e1e7ed), linear-gradient(#fff, #fff);
+  background-origin: border-box;
+  background-clip: content-box, border-box;
+}
+
+.search-form__input_big::placeholder {
+  height: 22px;
+  width: 129px;
+}
+
+.search-form__input_small::placeholder {
+  height: 20px;
+  width: 167.29px;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 19px;
+  padding-left: 0;
+  margin-left: -5px;
+}
+
+.search-form_small .search-form__label:after {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 1px;
+}
+
+.search-form__label_small:after {
+  left: 13px;
+  transform: translateY(6px);
+}
+
+.search-form__input_small {
+  padding-left: 30px;
+  padding-top: 3px;
```

Comment:
Not the right value. It looks okay because you've set the height for the placeholder. But it's incorrect to do so.